### PR TITLE
data directory passed in through global variable

### PR DESCRIPTION
**Pull request by ben-gready**
ok, changed to use this method: https://github.com/rstudio/shiny/issues/440

### CODE REVIEW

File: inst/SurveyAnalysisWebApp/app.R [hunk 1]
```diff
@@ -18,8 +18,8 @@ library(ggplot2)
 #source('/home/rstudio/code/inst/SurveyAnalysisWebApp/Modules.R')
 
 user <- "common" # this would eventually be an environment variable that defines the logged in user
-data_root <- Sys.getenv("NLP_DATA_PATH")
-store_loc = paste0(data_root, "/watson_nlp_responses/")
+#data_root <- Sys.getenv("NLP_DATA_PATH")
```
Comment (reisner) [inline inst/SurveyAnalysisWebApp/app.R]: @ben-gready delete commented stuff please

File: R/RunApp.R [hunk 2]
```diff
@@ -1,5 +1,7 @@
 #' @export
 runTextAnalyzeApp <- function(data_directory = "data/") {
+  .GlobalEnv$.nlp_data_path <- data_directory
+  on.exit(rm(.nlp_data_path, envir=.GlobalEnv))
   Sys.setenv("NLP_DATA_PATH" = data_directory)
```
Comment (reisner) [inline R/RunApp.R]: @ben-gready No longer need to set this

**Comment by ben-gready** [comment]
thanks for those comments - I'm missing lots of things today!  I like Pull Requests.

**Comment by ben-gready** [comment]
I removed the offending lines and am merging

**Pull request merged**